package.json

```diff
@@ -45,7 +45,7 @@
     "serve": "http-server build/",
     "lint": "eslint src",
     "fmt": "prettier --write configure *.js $(find src -name '*.jsx') $(find src -name '*.js') $(find tasks -name '*.js')",
-    "build": "./configure && ninja",
+    "build": "./configure && ninja -j 4",
```

Comment:
Why are you adding this? Ninja figures out how many processes to start based on your CPU count.

Reply:
My machine froze multiple times building it! and I had to restart it.

Reply:
Oh, I see.